### PR TITLE
update for solution content and social media content

### PR DESCRIPTION
**Pull request by abdulfahadlee**

### CODE REVIEW

File: assets/_templates/_header.php [hunk 1]
```diff
@@ -18,120 +18,109 @@
...
-                                                <li><a href="../../PRODUCTS.php#CRM" class="dropdown-item" type="button">Delivery Management</a></li>
+                                                <li><a href="../../products.php#CRM" class="dropdown-item" type="button">Delivery Management</a></li>
                                             </div>
                                             <div class="dropdown">
-                                                <li><a href="../../PRODUCTS.php#CRM" class="dropdown-item" type="button">Accounts Management</a></li>
+                                                <li><a href="../../products.php#CRM" class="dropdown-item" type="button">Accounts Management</a></li>
                                             </div>
                                             <div class="dropdown">
-                                                <li><a href="../../PRODUCTS.php#CRM" class="dropdown-item" type="button">Real Management</a></li>
+                                                <li><a href="../../products.php#CRM" class="dropdown-item" type="button">Real Management</a></li>
```
Comment (dkkaran1996) [inline assets/_templates/_header.php:42]: Its Real Estate Management

**Review by dkkaran1996 — CHANGES_REQUESTED** [review]
1. Some links shows different # and product are different in header.php
2. price.php file marking to PRODUCTS.php

File: assets/_templates/_price.php [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+                            Solution
+                        </a>
+                        <ul class="dropdown-menu" aria-labelledby="dropdownMenu2">
+                            <div class="dropdown">
+                                <div class="dropdown">
+                                    <li><a class="dropdown-item" type="button">Cloud</a></li>
+                                </div>
+                                <ul class="dropdown-menu" aria-labelledby="dropdownMenu2">
+                                    <div class="dropdown">
+                                        <li><a href="../../PRODUCTS.php" class="dropdown-item" type="button">Call Center Dialer</a></li>
```
Comment (dkkaran1996) [inline assets/_templates/_price.php:24]: It should be products.php instead of PRODUCTS.php

**Review by dkkaran1996 — CHANGES_REQUESTED** [review]

File: assets/_templates/_header.php (hunk 1 above)
Comment (abdulfahadlee) [reply assets/_templates/_header.php:42]: product details need real estate managment content for webpage.

**Pull request closed without merging**